smithy-swift-codegen/src/test/kotlin/software/amazon/smithy/swift/codegen/manifestanddocs/PackageManifestGeneratorTests.kt

```diff
@@ -10,6 +10,7 @@ import io.kotest.matchers.string.shouldStartWith
 import org.junit.jupiter.api.Assertions.assertNotNull
 import org.junit.jupiter.api.Test
 import software.amazon.smithy.swift.codegen.PackageManifestGenerator
+import software.amazon.smithy.swift.codegen.SwiftDependency
```

Comment:
Added a dependency to the manifest in this test so that more of the manifest can be checked (package & target dependencies.)